### DISCUSSION TITLE
Problems with Katex rending

### DISCUSSION THREAD

**Discussion by mpf** [marp-team/marp#405] [Q&A] [answered]
I'm observing different renderings for slides served from GitHub pages and by the Marp local server (eg, `marp -s`). In particular, here are screenshots of [this slide](https://friedlander.io/ubc-cpsc-406/slides/introduction#6). I'm using the math option `katex`. The source that slide is [here](https://github.com/mpf/ubc-cpsc-406/blob/main/slides/introduction.md#abstract-problem).

Any suggestions for what might be the problem?

## GitHub pages
![Screenshot 2023-01-10 at 4 53 43 PM](https://user-images.githubusercontent.com/109942/211692964-5f4a47f0-93b1-45a9-be6d-c3605c2c990a.png)

## Local server
![Screenshot 2023-01-10 at 4 53 53 PM](https://user-images.githubusercontent.com/109942/211692971-9f38966f-e424-4aca-9700-e86970b7047e.png)

**Comment by yhatt** [comment, marked as answer]
It does not reproduce in my latest Marp CLI. Which version are you using? Please share the result output of `marp --version`.

**Reply by mpf** [reply]
Thanks for the question about the version. Locally, I'm using v2.3.0, but for the Github action, I was using v1.7.0. I've since updated that version, and all is now working as it should! 💯

**Reply by yhatt** [reply]
Okay. Marked as answer :)
